### DISCUSSION TITLE
sections for ingredients

### DISCUSSION THREAD

**Discussion by martinmolt** [mealie-recipes/mealie#899] [Feature Request]
**Before submitting this feature request I have**
- [X] Reviewed the documentation and verified that my feature request isn't already a feature.
- [X] Reviewed the existing feature requests and my feature has not been requested.
- [X] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
There are no sections within the ingredients tab. In some recipes the ingredients are split up in 2 or more secions (e.g. bottom and filling for a cake). 

**(Optional): Suggest A Solution**
Implementing sections within the ingredients tab as seen in instructions tab.

If there are multiple solutions, please present each one separately. Save comparisons for the very end.)

**Additional Information**
I know it might be personal preference whether to have all ingredients combined to one list or split up like in some recepies like mentioned above. But the ability to choose between secions or no secions would be nice (I guess no priority tho).
- [ ] If the feature is accepted I'm willing to submit a PR to provide this feature (haven't looked into the code yet, but I guess you could maybe just copy the function of sections from the instructions tab)

Also thanks for your work, love it!

**Comment by hay-kot** [comment]
This has already been implemented. There's a plus button on an ingredient to add a section.

See here https://demo.mealie.io/recipe/rindergeschnetzeltes-in-pfefferrahm

**Reply by martinmolt** [reply]
Oh wow, haven't seen that. Thanks :)

**Reply by DennisGaida** [reply]
Also haven't seen this and found the feature via this request. The UI is a bit weird for the section: You click on the plus-button and then have to add a "first" ingredient to that section - why not make the whole line a "section line" without any ingredient?

Awesome feature!

**Reply by hay-kot** [reply]
> Also haven't seen this and found the feature via this request. The UI is a bit weird for the section: You click on the plus-button and then have to add a "first" ingredient to that section - why not make the whole line a "section line" without any ingredient?
> 
> Awesome feature!

I think that would have made the html templates and some of the backend processes more complicated. Thus seemed like the easiest approach.

**Reply by coolkau** [reply]
I also did not realize it exists. it is a bit hidden. And indeed it is a weird UI, but it works, if that is the easiest way to implement it :)
